src/nifreeze/utils/iterators.py

```diff
@@ -138,14 +149,17 @@ def centralsym_iterator(size: int | None = None, **kwargs) -> Iterator[int]:
 
     Parameters
     ----------
-    size : :obj:`int`
-        Number of volumes in the dataset
-        (for instance, the number of orientations in a DWI)
-
-    Returns
-    -------
-    :obj:`~typing.Iterator`
-        The sorted index order.
+    size : :obj:`int` or ``None`` (optional)
+        Number of volumes in the dataset.
+    bvalues : :obj:`list` (optional, keyword argument)
```

Comment:
Same comment as above about the style to document a kwarg.

Reply:
```suggestion
    bvalues : :obj:`list`, optional (keyword argument)
```